src/wallet/components/AcknowledgeX.tsx

```diff
@@ -1,26 +1,30 @@
 import React from 'react';
 import Button from 'reactstrap/lib/Button';
-import walletIcon from '../../../images/wallet_icon.svg';
-import SidebarLayout from '../SidebarLayout';
+import walletIcon from '../../images/wallet_icon.svg';
+import SidebarLayout from './SidebarLayout';
 
 interface Props {
-  challengeResponseAcknowledged: () => void;
+  title: string;
+  action: () => void;
+  actionTitle: string;
+  description: string;
 }
 
-export default class AcknowledgeChallengeResponse extends React.PureComponent<Props> {
+export default class AcknowledgeX extends React.PureComponent<Props> {
```

Comment:
I wonder if we should call these `Acknowledge` and move them into a subfolder to indicate they're generic components instead of `AcknowledgeX`?